### DISCUSSION TITLE
Map indices from line graph back to the original graph

### DISCUSSION THREAD

**Discussion by lisiq** [pyg-team/pytorch_geometric#5554] [Q&A] [answered]
Hi everyone!
I am using [LineGraph](https://pytorch-geometric.readthedocs.io/en/latest/_modules/torch_geometric/transforms/line_graph.html#LineGraph), and I was wondering if there is a way to map back from node indices of the line graph to node indices of the nodes in the original graph (ie. index of node v in the line graph maps to the node indices of u and w respectively)?
Thank you in advance!

**Comment by EdisonLeeeee** [comment, marked as answer]
The nodes in the line graph were mapped to the `edge_index` of the original graphs, so you can simply run:
```python
# v: index of node in the line graph
# u, w: index of node in the original graph
# edge_index: edge indices of original graph

u, w = edge_index[v]
```

**Reply by lisiq** [reply]
Thank you!
